### ISSUE TITLE
training, epochs and iteration

### ISSUE THREAD

**Issue by arantzadev** [grig-guz/rst-coref#5]
Hi 
I had a question concerning the training, is it normal that the there are about 6000 iterations for each epochs ?

I am only training on the rst-dt train set with an NVIDIA RTX A6000 and it takes about an hour for a single epoch.

Also why is the the current number of epochs set at 20 ?

Thank you

**Comment by grig-guz** [comment]
Yes, it is normal. There are ~300 trees but since we have a shift-reduce parser, each shift-reduce action is treated as an individual datapoint. So there are way more datapoints than trees.
We tried 10 and 15 epochs but the validation scores didn't saturate. They didn't improve beyond 20.
